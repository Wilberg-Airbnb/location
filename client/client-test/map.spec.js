import React from 'react';
import renderer from 'react-test-renderer';
import Map from '../components/map.jsx';


import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
 
configure({ adapter: new Adapter() });
 
describe('Renders Components', () => {
  let component;
  beforeEach( () => {
    component = shallow(<Map lat={34.052235} lng={-118.243683}/>)
  });

  test("should render initial layout", () => {
    const component = shallow(<Map />);
    expect(component.getElements()).toMatchSnapshot();
  });

  test("should create a map in component state", ()=> {
    expect(component.contains(<Map />)).toBe(false)
  });

 
});


