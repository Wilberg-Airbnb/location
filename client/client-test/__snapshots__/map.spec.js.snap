// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Components should render initial layout 1`] = `
Array [
  <div
    id="map"
    style={
      Object {
        "height": "300px",
        "width": "400px",
      }
    }
  />,
]
`;
